block-producer/src/test_utils/block.rs

```diff
@@ -29,8 +31,7 @@ pub async fn build_expected_block_header(
         let mut store_accounts = store.accounts.read().await.clone();
         for (account_id, new_account_state) in updated_accounts.iter() {
             store_accounts
-                .update_leaf(u64::from(*account_id), new_account_state.into())
-                .unwrap();
+                .insert(LeafIndex::new_max_depth(u64::from(*account_id)), new_account_state.into());
```

Comment:
I've implemented `From<AccountId> for `LeafIndex<64>` - so, we should be able to do `account_id.into()` here instead of `(LeafIndex::new_max_depth(u64::from(*account_id))`.

Reply:
fixed